ProjectWar/GameException.h

```diff
@@ -33,5 +33,20 @@ class InvalidPositionException : public std::runtime_error {
     int width;
     int height;
 };
+        
+class IllegalStateException : public std::runtime_error {
+public:
+    IllegalStateException( std::string msg )
```

Comment:
for that i think i'm going to make a user story to change it for all the custom exceptions